.github/workflows/Spack.yaml

```diff
@@ -0,0 +1,76 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Regularly scheduled installation test with the Spack package manager
+name: Spack
+
+on:
+  schedule:
+    - cron: '0 0 * * 1' # every Monday morning
+  workflow_dispatch:
+
+concurrency:
+  group: spack
+  cancel-in-progress: true
```

Comment:
Just for my understanding, this means that if another spack installation group is running, it will cancel those? Or would it cancel a PR merge?

Reply:
The former. It means (1) only one job running this workflow can run at a time, and (2) if another job for this workflow is triggered (by the `schedule` or `workflow_dispatch` events listed above), then the previous job is canceled. I did this because the workflow doesn't depend on code in the repo, so it's useless to run it multiple times in parallel.